src/graph/transform/cuda/knn.cu

```diff
@@ -22,6 +23,14 @@
 namespace dgl {
 namespace transform {
 namespace impl {
+
+template <typename Type>
+static __host__ __device__ std::enable_if_t<std::is_unsigned<Type>::value, Type>
+Pow2Align(Type size, Type align) {
+  if (align <= 1 || size <= 0) return size;
+  return ((size - 1) | (align - 1)) + 1;
```

Comment:
could you add some example here to help reader understand what aligned value will be returned here? such as `Pow2Align(17, 4) -> 20`, `(17, 8) -> 24`?

Reply:
sure